### PR TITLE
feat: Accounts, Selling & Assets Onboarding cleanup

### PR DESCRIPTION
**Pull request by kamaljohnson**
feat(onboarding-cleanup): Add onboarding steps for **Accounting**, **Assets** and **Selling** modules

 ### Accounting
 - **Review Company** _( show 'Company List' )_
 
image

 - **Review Chart of Accounts** _( show 'Chart of Accounts' )_

image

 - **Setting up Taxes** _( form tour for 'Sales Taxes and Charges Template' )_

image 

 - **Account Settings** _( form tour for 'Account Settings' )_

image

 - **Cost Centers for Budgeting and Analysis** _( show 'Cost Center Tree' )_

image

 - **Create your first Purchase Invoice** _( form tour for 'Purchase Invoice' )_

image

 - **Updating Opening Balances** _(  video tutorial )_ 

image

 - **Financial Statements** _( show general report )_

image
   
 ### Assets
 - **Review Fixed Asset Accounts** _( show 'Chart of Accounts' )_

image 

 - **Define Asset Category** _( form tour for 'Asset Category' )_
 
image

 - **Create Asset Item** _(  form tour for 'Item' )_

image

 - **Purchase an Asset** _( form tour for 'Purchase Receipt' )_

image

 - **Add an Existing Asset** _( form tour for 'Asset' )_

image

### Selling
 - **Selling Settings** _( form tour for 'Selling Settings' )_
 
image

- **Sales Order** _( form tour for 'Sales Order' )_

image

no-docs

### CODE REVIEW

File: erpnext/accounts/form_tour/purchase_invoice/purchase_invoice.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+   "description": "Add Items in the table",
+   "field": "",
+   "fieldname": "items",
+   "fieldtype": "Table",
+   "has_next_condition": 0,
+   "is_table_field": 0,
+   "label": "Items",
+   "parent_field": "",
+   "position": "Bottom",
+   "title": "List Of Items"
```
Comment (nabinhait) [inline erpnext/accounts/form_tour/purchase_invoice/purchase_invoice.json]:
```suggestion
   "title": "List of Items"
```

File: erpnext/accounts/form_tour/purchase_invoice/purchase_invoice.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   "has_next_condition": 1,
+   "is_table_field": 0,
+   "label": "Supplier",
+   "next_step_condition": "supplier",
+   "parent_field": "",
+   "position": "Right",
+   "title": "Select Supplier"
+  },
+  {
+   "description": "Add Items in the table",
```
Comment (nabinhait) [inline erpnext/accounts/form_tour/purchase_invoice/purchase_invoice.json]:
```suggestion
   "description": "Add items in the table",
```

File: erpnext/accounts/form_tour/sales_taxes_and_charges_template/sales_taxes_and_charges_template.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+   "description": "Set this template as the default for all sales transactions",
+   "field": "",
+   "fieldname": "is_default",
+   "fieldtype": "Check",
+   "has_next_condition": 0,
+   "is_table_field": 0,
+   "label": "Default",
+   "parent_field": "",
+   "position": "Bottom",
+   "title": "Is this Default?"
```
Comment (nabinhait) [inline erpnext/accounts/form_tour/sales_taxes_and_charges_template/sales_taxes_and_charges_template.json]:
```suggestion
   "title": "Is this Default Tax Template?"
```

File: erpnext/accounts/onboarding_step/financial_statements/financial_statements.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+ "action": "View Report",
+ "creation": "2021-07-12 12:08:47.026115",
+ "description": "# Financial Statements\n\nIn ERPNext, you can get crucial financial reports like [Balance Sheet] and {Profit and Loss] statements with a click of a button. You can run in the report for a different period and plot analytics charts premised on statement data. For more reports, check sections like Financial Statements, General Ledger, and Profitability reports.\n\n<b>[Check Accounting reports](https://docs.erpnext.com/docs/v13/user/manual/en/accounts/accounting-reports)</b>",
```
Comment (nabinhait) [inline erpnext/accounts/onboarding_step/financial_statements/financial_statements.json]:
```suggestion
 "description": "# Financial Statements\n\nIn ERPNext, you can get crucial financial reports like [Balance Sheet] and [Profit and Loss] statements with a click of a button. You can run in the report for a different period and plot analytics charts premised on statement data. For more reports, check sections like Financial Statements, General Ledger, and Profitability reports.\n\n<b>[Check Accounting reports](https://docs.erpnext.com/docs/v13/user/manual/en/accounts/accounting-reports)</b>",
```

File: erpnext/assets/form_tour/asset_category/asset_category.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+ "modified": "2021-08-24 12:48:20.763173",
+ "modified_by": "Administrator",
+ "module": "Assets",
+ "name": "Asset Category",
+ "owner": "Administrator",
+ "reference_doctype": "Asset Category",
+ "save_on_complete": 0,
+ "steps": [
+  {
+   "description": "Name Asset category. You can create categories based on Asset Types. like Furniture, Property, Electronics etc.",
```
Comment (nabinhait) [inline erpnext/assets/form_tour/asset_category/asset_category.json]:
```suggestion
   "description": "Name Asset category. You can create categories based on Asset Types like Furniture, Property, Electronics etc.",
```

File: erpnext/assets/onboarding_step/existing_asset/existing_asset.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
+ "action": "Show Form Tour",
+ "action_label": "Add an existing asset item",
```
Comment (nabinhait) [inline erpnext/assets/onboarding_step/existing_asset/existing_asset.json]:
```suggestion
 "action_label": "Add an existing Asset",
```

File: erpnext/assets/onboarding_step/existing_asset/existing_asset.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+ "creation": "2021-08-13 14:28:30.650459",
+ "description": "# Add an Existing Asset\n\nIf you are just starting with ERPNext, you will need to enter Assets you already possess. You can add them as opening fixed assets in ERPNext. Please note that you will have to make a Journal Entry separately updating the opening balance in the fixed asset account.",
```
Comment (nabinhait) [inline erpnext/assets/onboarding_step/existing_asset/existing_asset.json]:
```suggestion
 "description": "# Add an Existing Asset\n\nIf you are just starting with ERPNext, you will need to enter Assets you already possess. You can add them as existing fixed assets in ERPNext. Please note that you will have to make a Journal Entry separately updating the opening balance in the fixed asset account.",
```

File: erpnext/selling/doctype/sales_order/sales_order.js [hunk 8]
```diff
@@ -102,6 +102,38 @@ frappe.ui.form.on("Sales Order Item", {
 	}
 });
 
+frappe.tour['Sales Order'] = [
```
Comment (nabinhait) [inline erpnext/selling/doctype/sales_order/sales_order.js]: Remove this

File: erpnext/stock/form_tour/item/item.json [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+   "fieldtype": "Check",
+   "has_next_condition": 0,
+   "is_table_field": 0,
+   "label": "Is Fixed Asset",
+   "parent_field": "",
+   "position": "Bottom",
+   "title": "Is this a Fixed Asset?"
+  },
+  {
+   "description": "Check Auto Create Asset on Purchase",
```
Comment (nabinhait) [inline erpnext/stock/form_tour/item/item.json]:
```suggestion
   "description": "On checking it, the system will create an Asset automatically on purchase",
```

File: erpnext/stock/form_tour/item/item.json [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+   "fieldtype": "Link",
+   "has_next_condition": 0,
+   "is_table_field": 0,
+   "label": "Asset Category",
+   "parent_field": "",
+   "position": "Bottom",
+   "title": "Asset Category"
+  },
+  {
+   "description": "Select a naming series",
```
Comment (nabinhait) [inline erpnext/stock/form_tour/item/item.json]:
```suggestion
   "description": "Select a naming series which will be used to create an Asset automatically",
```

**Review by nabinhait — CHANGES_REQUESTED** [review]

**Comment by nabinhait** [comment]
@kamaljohnson Errors while installing fixtures

**Comment by kamaljohnson** [comment]
> @kamaljohnson Errors while installing fixtures

@nabinhait the local site is running fine for me, Can you please show the error.

**Comment by nextchamp-saqib** [comment]
@mergifyio backport version-13-hotfix

**Comment by nextchamp-saqib** [comment]
@mergifyio backport version-13-pre-release

**Pull request merged**